### DISCUSSION TITLE
Change in MDOC path detected when it wasn't changed

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#54] [General]
I noticed that when rerunning a block, a change is detected in the path to MDOC files even when the path has not been changed by the user.

`3	2024-02-03 14:01:49 [INFO] pyp/system/project_params.py:784 | Parameters changed: {'tomo_rec_topaz_denoise', 'movie_weights', 'slurm_merge_memory', 'data_path_mdoc', 'slurm_gres', 'slurm_merge_walltime'}`

Is this by design? If not, is it possible that rerunning the block will cause unnecessary reprocessing of the tilt series?

**Comment by abartesaghi** [comment]
The log lines right after that should explicitly say what is being recalculated and due to what parameter changes. In general, changes in filesystem paths can be mistakenly detected due to the way the directories are stored in the project configuration file, but these should not trigger recalculation of any unnecessary pre-processing steps.
